modules/op-running-a-pipeline.adoc

```diff
@@ -90,4 +90,4 @@ $ tkn pipeline start build-and-deploy --last
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../pipelines/authenticating-pipelines-using-git-secret.adoc#authenticating-pipelines-using-git-secret[Authenticating pipelines using git secret]
+* xref:../secure/authenticating-pipelines-using-git-secret.adoc#authenticating-pipelines-using-git-secret[Authenticating pipelines using git secret]
```

Comment:
@mramendi No xrefs in modules. Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

Reply:
Again, not in scope.